formbricks/formbricks
# feat: Google Tag Integration

## What does this PR do?



Feat: add new integration option for google tag manager for formbricks survey.

[7f436137-67e7-4a3e-a043-cee0ef22aaa8.webm](https://github.com/formbricks/formbricks/assets/42396310/c77c4c7b-5bf2-4453-81b4-f8c5d2f3f603)

## How should this be tested?



- Add new tag then try to verify if the tag is already called when open the survey
- Since each survey can have multiple tags, you must verify that the survey should have the right tag.

## Checklist



### Required


### Appreciated
